csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt

```diff
@@ -63,12 +63,12 @@ sf-003/parquet/15/part.0.parquet
 sf-003/parquet/15/part.1.parquet
 
 KATANA_DEFAULT_LABEL=KNOWS
-creationDate:IGNORE|:START_ID(Person)|:END_ID(Person)
+creationDate:DATETIME|:START_ID(Person)|:END_ID(Person)
 sf-003/parquet/16/part.0.parquet
 sf-003/parquet/16/part.1.parquet
 
 KATANA_DEFAULT_LABEL=
-creationDate:IGNORE|:START_ID(Person)|:END_ID(Post)
+creationDate:DATETIME|:START_ID(Person)|:END_ID(Post)
```

Comment:
what are edges between Person and Post?  I do not see that in the spec

Reply:
Every Post is also a Message (it's a derived type). This edge is to indicate that a person "likes" a post.